Kinvey-Xamarin/Store/DataStore.cs

```diff
@@ -101,6 +101,9 @@ public void SetCustomRequestProperty (string key, JObject value)
 			this.customRequestProperties.Add (key, value);
 		}
 
+		KinveyRealtimeDelegate<T> RealtimeCallback { get; set; }
+		KinveyRealtimeDelegate<string> routerCallback;
```

Comment:
Why does the `DataStore` need to keep a reference to the `routerCallback`? Shouldn't it set it and then forget about it?